README.md

```diff
@@ -42,7 +42,7 @@ The 3D view in Design Studio is just a video stream from our hosted geometry eng
 
 We recommend downloading the latest application binary from our [releases](https://github.com/KittyCAD/modeling-app/releases) page. If you don't see your platform or architecture supported there, please file an issue.
 
-If you'd like to try out upcoming changes sooner, you can also download those from our [nightly releases](https://zoo.dev/modeling-app/download/nightly) page.
+If you'd like to try out upcoming changes sooner, you can also download those from our [staging releases](https://zoo.dev/modeling-app/download/staging) page.
```

Comment:
If this build points to the "dev" infrastructure, will customers be able to use it? Perhaps we want both:

- Nightly: customer-facing and uses "prod" infrastructure
- Staging: internal-only and uses the "dev" (eventually named "staging") infrastructure

Reply:
Yeah this line has to go. Reason I'm doing this right now is that an internal-only, dev-infra, commit-by-commit deployment could be extremely useful next week to get validation on things fast

I think we don't really need it now but I can get behind the idea of a Beta channel that's public facing and gets new features quicker, pretty much what we discussed at https://kittycadworkspace.slack.com/archives/C07A80B83FS/p1746797055394379?thread_ts=1746540003.331569&cid=C07A80B83FS

Reply:
Yeah, removing this line works for me! Do we need to hide/modify https://zoo.dev/modeling-app/download/nightly or just accept that customers can't use it while we get v1 out?